app/dashboard/models.py

```diff
@@ -191,8 +191,12 @@ class Bounty(SuperModel):
     balance = models.DecimalField(default=0, decimal_places=2, max_digits=50)
     accepted = models.BooleanField(default=False, help_text=_('Whether the bounty has been done'))
     interested = models.ManyToManyField('dashboard.Interest', blank=True)
+    opened_comment = models.IntegerField(null=True, blank=True)
     interested_comment = models.IntegerField(null=True, blank=True)
+    started_comment = models.IntegerField(null=True, blank=True)
+    stopped_comment = models.IntegerField(null=True, blank=True)
     submissions_comment = models.IntegerField(null=True, blank=True)
+    done_comment = models.IntegerField(null=True, blank=True)
```

Comment:
can you make migrations with `./manage.py makemigrations` and then deploy them to this PR?

Reply:
Don't we want only one github comment ID field here? We could simply patch the comment or delete/create a new one as shown in the design? Also, you can run `make migrations` to execute directly against the docker container.

Reply:
@mbeacom I kept one comment for each state in the flow of the design. we could collapse them into one field using `from django.contrib.postgres.fields import `ArrayField`

Reply:
Why do we need to know about past comments other than the last patched ID?  We don't want a new gitcoin bot comment for every state change type, do we?

Reply:
@mbeacom I don't see that in the requirements actually, so I assumed one comment per state in the flow would be required and each one would be edited appropriately as the issue cycles through states in the flow. That was what I was gathering from making things clearer, however the requirement may be to post each new change.
I think @PixelantDesign had some ideas on how this should work, i.e. maybe all new information should be presented in new comments. What do you guys thing?

Reply:
I have spoken with @PixelantDesign about this. We'll want to consolidate to use only one comment_id and simply patch the comment and update the singular comment ID field when necessary.  State changes are reflected in the comment edit history.

Reply:
@mbeacom Done!